planus-cli/src/codegen/templates/rust/struct.template

```diff
@@ -88,8 +88,8 @@ impl<'a> ::core::fmt::Debug for {{ info.ref_name }}<'a> {
         let mut f = f.debug_struct("{{info.ref_name}}");
         {% for field in fields -%}
             {%- if field.info.getter_return_type.starts_with("planus::Result<") -%}
-                if let ::core::option::Option::Some(value) = self.{{field.info.name}}().transpose() {
-                    f.field("{{field.info.name}}", &value);
+                if let ::core::option::Option::Some(field_{{field.info.name}}) = self.{{field.info.name}}().transpose() {
+                    f.field("{{field.info.name}}", &field_{{field.info.name}});
```

Comment:
Is this just for consistency? As far as I can see it doesn't make a difference.

Reply:
Yes, this is for consistency.